env_shell_python_git/environment.yml

```diff
@@ -0,0 +1,8 @@
+name: swc-shell-python-git
+channels:
+  - swc
+dependencies:
+  - swc-shell
+  - swc-git
+  - swc-python
+  - python=2.7.*
```

Comment:
I was a bit surprised to code that, myself.  I did so based on the workshop page for my pending workshop http://jiarong.github.io/2015-11-05-ND/ which states that Python should be 2.7, not 3.x

What is the current state of teaching Py3 instead?  Is this just something in the project template that needs updating, or did my partner either clone an old template, or choose py27 instead?  I guess I should ask him directly.